code/modules/admin/verbs/request_music.dm

```diff
@@ -0,0 +1,29 @@
+/mob/verb/request_music()
+	set category = "OOC"
+	set name = "Request Music"
+
+	if(GLOB.say_disabled) //This is here to try to identify lag problems
+		to_chat(usr, span_danger("Speech is currently admin-disabled."), confidential = TRUE)
+		return
+
+	var/msg = tgui_input_text(usr.client, "Please Input a URL", "Music Request", "")
```

Comment:
This should be named something like "request_url" or something. msg makes no sense.